### ISSUE TITLE
Custom validation rules in DVR do not seem to register properly

### ISSUE THREAD

**Issue by d4rky-pl** [foxhound87/mobx-ajv-form#278]
Minimal test case:

```
import validatorjs from 'validatorjs'
import MobxReactForm from 'mobx-react-form'

const dvr = {
  package: validatorjs,
  extend: (v) => {
    console.log('During extend')
    v.register(
      'some_rule',
      function() { console.log('When validating'); return false },
      'yay, fail!'
    )
  }
}

const fields = ['name']
const rules = {'name': 'some_rule'}

const myForm = new MobxReactForm({ fields, rules }, { plugins: { dvr }})
myForm.validate().then(({ isValid }) => console.log('Validation result: ', isValid))
```

Expected:

```
During extend
When validating
Validation result:  false
```

Actual:

```
During extend
Validation result:  true
```

Sanity check to make sure validatorjs is not the culprit:

```
validatorjs.register(
  'some_rule',
  function() { console.log('When validating'); return false },
  'yay, fail!'
)

const validation = new validatorjs({ name: 'foo' }, { name: 'some_rule' })
console.log("Validation result: ", validation.passes())

// Correctly returns:
// When validating
// Validation result:  false
```

**Comment by foxhound87** [comment]
I will check this asap, thank you for the detailed test case!

**Comment by foxhound87** [comment]
Notice that adding `const values = { name: 'foo' };` to the form, the validation function **will be executed**. If you change `{ name: 'foo' }` to `{ name: '' }` from your **Sanity check** example and the function is not exectued then it's a validatorjs behavior. Can you confirm this? If so we should open an issue there.

**Comment by d4rky-pl** [comment]
I can confirm that when the `name` is empty, validatorjs does not run custom validations. Diving deeper into their GH issues unfortunately proves that it's by design and will not be fixed - skaterdav85/validatorjs#118 . Workaround is to add `required` rule.

I'm closing this issue as it has nothing to do with mobx-react-form.
